README.md

```diff
@@ -115,14 +127,41 @@ For requesting the actor to the server you have to start the tracker. This way a
 TrackerAsset.Instance.Start();
 ```
 
+### Tracker Login and Start (asynchronous)
+
+To use the tracker asynchronously, you should use the methods LoginAsync and StartAsync.
```

Comment:
To use the tracker asynchronously you have to enable the ASYNC precompiler tag (as you specified in line 39 of this document). This way, traces will be sent asynchronously in any case. However, for coding simplicity, Start and Login have been maintained both sync and async (this last one will only work if tracker has ASYNC tag enabled). Depending on your code you might want to use one or the other.